### PR TITLE
urxvt: add page

### PR DESCRIPTION
**Pull request by Alone2**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/urxvt.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# urxvt
+
+> Rxvt-unicode.
+> A customizable terminal emulator.
+
+- Open a new urxvt window:
+
+`urxvt`
+
+- Run in a specific working directory:
```
Comment (agnivade) [inline pages/linux/urxvt.md]: We can remove the word "working"

File: pages/linux/urxvt.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`urxvt -cd {{directory}}`
```
Comment (agnivade) [inline pages/linux/urxvt.md]: Let's change directory to path/to/directory

File: pages/linux/urxvt.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`urxvt -e {{command}}`
+
+- Run a command and keep the window open:
+
+`urxvt --hold -e {{command}}`
+
+- Run a command within the "sh" shell:
+
+`urxvt -e sh -c {{command}}`
```
Comment (agnivade) [inline pages/linux/urxvt.md]: `sh` should be in tokens (`{{}}`)

File: pages/linux/urxvt.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`urxvt`
+
+- Run in a specific working directory:
+
+`urxvt -cd {{directory}}`
+
+- Run a command in a new urxvt window:
+
+`urxvt -e {{command}}`
```
Comment (agnivade) [inline pages/linux/urxvt.md]: Is there an expanded version of `e` ? Right now, it does not map to any word.

Comment (Alone2) [reply pages/linux/urxvt.md]: Sadly, there is no expanded version of "-e".

**Pull request merged**